notebook/tune_lexicographic.ipynb

```diff
@@ -0,0 +1,285 @@
+{
```

Comment:
Add a page under webiste/docs/Examples which has a pointer in the end to this notebook, like other examples. The page doesn't need to contain everything in this notebook.